_posts/2017-10-20-jbhc17-from-my-perspective.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title:  "#jbhc17 from my perspective"
+date: 2017-10-20 14:03:00 +0200
+categories: meetup
+author_name: Jacek Smolak
+author_url : /author/jaceksmolak
+author_avatar: jaceksmolak
+show_avatar : true
+read_time : 4
+show_related_posts: false
+square_related:
+feature_image: posts/2017-10-22-jbhc17-from-my-perspective/poster.jpg
+---
+
+When I first heard about [JB][jbrains] coming to HolidayCheck to work with us I was excited.
+I haven't heard much about him then, so I began googling him. I did remember the [7-26-talk]
+(which is awesome, you need to check it out), but I didn't know much more.
+Once I got to learn about him, I already knew - I can't miss this opportunity. And I didn't.
+
+## The trip
+
+As I am from Poland, I (actually me and my colleagues), had to hop on plane and get to Munich. Once there, we
+went to the hotel, went out for a burger and a beer, got back, went to sleep. Morning came.
+We arrived to meeting place around 9. JB was already there, preparing.
+I approached him, shook his hand, exchanged few words. Seems like a nice guy - I thought.
+
+## First day
+
+I sat down in 2nd row, so that I would not miss a thing. Keynote began. I'll be frank. From the moment he started,
+till the very end I listened to everything he had to say. Imagine his 7-26-talk, but for few hours (with breaks of course ;)).
+It was all very interesting. JB has this skill to attract peoples attention and talk in a very interesting way.
+Imagine a lesson you liked in school, because anything you heard there was valuable and attracted your attention.
+It was the same.
+
+The main reason I chose to go to [#jbhc17][jbhc17] was TDD and craftsmanship concerning our daily job.
```

Comment:
The next few sentences seem a bit too "conversational", "spoken word" to me

> "It was interesting, but still"
Still what?

As a suggestion:

The topics that I was most interested in hearing about at #JBHC17 were *TDD* and *Craftsmansip*. When these hardly came up during the first day I started to get a bit worried. It was all really interesting yet not what I was looking for. Little did I know, that JB had a plan: